client/css/style.css

```diff
@@ -1671,6 +1671,55 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	margin-top: 0.2em;
 }
 
+.password-container {
+	position: relative;
+}
+
+.password-container input {
+	padding-right: 37px;
+}
+
+#sign-in .password-container {
+	width: 100%;
+}
+
+#sign-in .password-container .reveal-password {
+	top: 31px;
+	right: 0;
+}
+
+.password-container .reveal-password {
+	height: 37px;
+	width: 37px;
+	position: absolute;
+	top: 2px;
+	right: 15px;
+	cursor: pointer;
```

Comment:
I would love if we could make the `.reveal-password` element a `button` instead of a `span`. It comes with benefits such as no need to set the cursor property, but also improves accessibility with keyboard navigation and such. Thoughts?

Reply:
I'm not sure it should be keyboard accessible (tabbing into it would be counter intuitive?).

Edge doesn't allow tabbing into their reveal button.